net/seafile-ccnet/Makefile

```diff
@@ -74,12 +73,9 @@ CONFIGURE_ARGS += \
 	--enable-python \
 	--with-mysql="$(STAGING_DIR)/usr/bin/mysql_config"
 
-# Include Python 2 bindings in this package to maintain compatibility with previous versions
 define Package/seafile-ccnet/install
 	$(INSTALL_DIR) $(1)/usr/{lib,libexec}
-	$(INSTALL_DIR) $(1)$(PYTHON_PKG_DIR)/ccnet
```

Comment:
should this be `PYTHON3_PKG_DIR`?
or do we need it?

Reply:
Similar to seafile-server, there is now a [python3-seafile-ccnet](https://github.com/openwrt/packages/blob/76bd346085ce50c05c27ea1d6fa1452e015f36d7/net/seafile-ccnet/Makefile#L56) package.

Reply:
ack